lib/sshutils/server.go

```diff
@@ -457,7 +457,10 @@ func (s *Server) HandleConnection(conn net.Conn) {
 			// send keepalive pings to the clients
 		case <-keepAliveTick.C:
 			const wantReply = true
-			sconn.SendRequest(teleport.KeepAliveReqType, wantReply, keepAlivePayload[:])
+			_, _, err = sconn.SendRequest(teleport.KeepAliveReqType, wantReply, keepAlivePayload[:])
+			if err != nil {
+				log.Errorf("failed sending keepalive request: %v", err)
```

Comment:
Missing capitalization and punctuation.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Fixed